CONTRIBUTING.md

```diff
@@ -0,0 +1,245 @@
+# How to contribute to OMZ
+
+From this document you will know how to contribute your model to OpenVINO&trade; Open Model Zoo. Almost any model from supported frameworks (see list below) can be added. It could be done in few simple steps.
+
+1. [Model location](#model-location)
+2. [Model conversion](#model-conversion)
+3. [Demo](#demo)
+4. [Accuracy validation](#accuracy-validation)
+7. [Configuration file](#configuration-file)
+6. [Documentation](#documentation)
+7. [Pull request requirements](#pull-request-requirements)
+
+List of supported frameworks:
+* Caffe\*
+* Caffe2\* (by conversion to ONNX\*)
+* TensorFlow\*
+* MXNet\*
+* PyTorch\* (by conversion to ONNX\*)
+
+## Model location
+
+Upload your model to any Internet file storage with easy and direct access to it. It can be www.github.com, GoogleDrive\*, or any other.
+
+*After this step you will get **links** to the model*
+
+## Model conversion
+
+OpenVINO&trade; supports models in its own format IR. Model from any supported framework can be easily converted to IR using Model Optimizer tool included in OpenVINO&trade; package. More information about conversion you can learn [here](https://docs.openvinotoolkit.org/latest/_docs_MO_DG_Deep_Learning_Model_Optimizer_DevGuide.html). After successful conversion you will get model in IR format `*.xml` representing net graph and `*.bin` containing net parameters. 
+
+> **NOTE 1**: due to OpenVINO&trade; paradigm, image pre-processing parameters (mean and scale) should be built into converted model to simplify model usage.
+
+> **NOTE 2**: due to OpenVINO&trade; paradigms, if model input is a colored image, color channel order should be `BGR`.
+
+*After this step you`ll get **conversion parameters** for Model Optimizer.*
+
+## Demo
+
+Demo will show main idea of how work with your model. If your model solves one of the supported by Open Model Zoo task, try find appropriate option from [demos](https://docs.openvinotoolkit.org/latest/_demos_README.html) or [samples](https://docs.openvinotoolkit.org/latest/_docs_IE_DG_Samples_Overview.html).
+
+If appropriate demo or sample are absent, you must provide your own demo (C++ or Python). Demo's input requires next options:
+```
+    -i "<path>"              Optional. Path to a input file or directory (for multiple inferences). By default input must be generated randomly.
```

Comment:
> By default input must be generated randomly.

Where did this come from? I don't think any of the existing demos do this.

Reply:
* `-i` is Required. I agree with @IRDonch 

* Let's avoid specifying what exactly should be an input, because it may depend on use case and it also usually depends on what is installed and can be used as `VideoCapture`'s backend. I would be OK with `Input to process`.